googletest/include/gtest/internal/gtest-port.h

```diff
@@ -2127,6 +2127,32 @@ struct IteratorTraits<const T*> {
   typedef T value_type;
 };
 
+// remove const and reference qualifier
```

Comment:
It would be preferable to use standard traits

Reply:
I've splitted `remove_constref` to `remove_const` and `remove_reference`. In addition, I removed the `remove_reference` in gmock-internal-util.h because it is duplicated.

Reply:
Please completely remove your own implementation and use https://en.cppreference.com/w/cpp/types/remove_reference and related

Reply:
Including `<type_traits>` requires C++11 compiler. Is it OK?

Reply:
Done.